<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="IN-L2504" ignored="0" name="Bill Paid" tests="11" failures="7" timestamp="18 May 2021 14:49:19 GMT" time="508.841" errors="0">
  <testcase name="testGetNotificationsForBillPaid" time="45.89" classname="com.yodlee.app.yodleeApi.Insights.Notifications.V2.TestGetInsightsForBillPaid"/>
  <testcase name="testGetNotificationsForBillPaid" time="46.148" classname="com.yodlee.app.yodleeApi.Insights.Notifications.V2.TestGetInsightsForBillPaid"/>
  <testcase name="testGetNotificationsForBillPaid" time="45.427" classname="com.yodlee.app.yodleeApi.Insights.Notifications.V2.TestGetInsightsForBillPaid">
    <failure type="java.lang.AssertionError" message="TEST FAILED">
      <![CDATA[java.lang.AssertionError: TEST FAILED
	at com.yodlee.app.yodleeApi.Insights.Notifications.V2.TestGetInsightsForBillPaid.testGetNotificationsForBillPaid(TestGetInsightsForBillPaid.java:350)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
... Removed 11 stack frames]]>
    </failure>
  </testcase> <!-- testGetNotificationsForBillPaid -->
  <testcase name="testGetNotificationsForBillPaid" time="46.599" classname="com.yodlee.app.yodleeApi.Insights.Notifications.V2.TestGetInsightsForBillPaid"/>
  <testcase name="testGetNotificationsForBillPaid" time="45.407" classname="com.yodlee.app.yodleeApi.Insights.Notifications.V2.TestGetInsightsForBillPaid">
    <failure type="java.lang.AssertionError" message="TEST FAILED">
      <![CDATA[java.lang.AssertionError: TEST FAILED
	at com.yodlee.app.yodleeApi.Insights.Notifications.V2.TestGetInsightsForBillPaid.testGetNotificationsForBillPaid(TestGetInsightsForBillPaid.java:350)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
... Removed 11 stack frames]]>
    </failure>
  </testcase> <!-- testGetNotificationsForBillPaid -->
  <testcase name="testGetNotificationsForBillPaid" time="45.622" classname="com.yodlee.app.yodleeApi.Insights.Notifications.V2.TestGetInsightsForBillPaid">
    <failure type="java.lang.AssertionError" message="TEST FAILED">
      <![CDATA[java.lang.AssertionError: TEST FAILED
	at com.yodlee.app.yodleeApi.Insights.Notifications.V2.TestGetInsightsForBillPaid.testGetNotificationsForBillPaid(TestGetInsightsForBillPaid.java:350)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
... Removed 11 stack frames]]>
    </failure>
  </testcase> <!-- testGetNotificationsForBillPaid -->
  <testcase name="testGetNotificationsForBillPaid" time="46.304" classname="com.yodlee.app.yodleeApi.Insights.Notifications.V2.TestGetInsightsForBillPaid">
    <failure type="java.lang.AssertionError" message="TEST FAILED">
      <![CDATA[java.lang.AssertionError: TEST FAILED
	at com.yodlee.app.yodleeApi.Insights.Notifications.V2.TestGetInsightsForBillPaid.testGetNotificationsForBillPaid(TestGetInsightsForBillPaid.java:350)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
... Removed 11 stack frames]]>
    </failure>
  </testcase> <!-- testGetNotificationsForBillPaid -->
  <testcase name="testGetNotificationsForBillPaid" time="45.615" classname="com.yodlee.app.yodleeApi.Insights.Notifications.V2.TestGetInsightsForBillPaid">
    <failure type="java.lang.AssertionError" message="TEST FAILED">
      <![CDATA[java.lang.AssertionError: TEST FAILED
	at com.yodlee.app.yodleeApi.Insights.Notifications.V2.TestGetInsightsForBillPaid.testGetNotificationsForBillPaid(TestGetInsightsForBillPaid.java:350)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
... Removed 11 stack frames]]>
    </failure>
  </testcase> <!-- testGetNotificationsForBillPaid -->
  <testcase name="testGetNotificationsForBillPaid" time="46.391" classname="com.yodlee.app.yodleeApi.Insights.Notifications.V2.TestGetInsightsForBillPaid">
    <failure type="java.lang.AssertionError" message="TEST FAILED">
      <![CDATA[java.lang.AssertionError: TEST FAILED
	at com.yodlee.app.yodleeApi.Insights.Notifications.V2.TestGetInsightsForBillPaid.testGetNotificationsForBillPaid(TestGetInsightsForBillPaid.java:350)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
... Removed 11 stack frames]]>
    </failure>
  </testcase> <!-- testGetNotificationsForBillPaid -->
  <testcase name="testGetNotificationsForBillPaid" time="45.938" classname="com.yodlee.app.yodleeApi.Insights.Notifications.V2.TestGetInsightsForBillPaid">
    <failure type="java.lang.AssertionError" message="TEST FAILED">
      <![CDATA[java.lang.AssertionError: TEST FAILED
	at com.yodlee.app.yodleeApi.Insights.Notifications.V2.TestGetInsightsForBillPaid.testGetNotificationsForBillPaid(TestGetInsightsForBillPaid.java:350)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
... Removed 11 stack frames]]>
    </failure>
  </testcase> <!-- testGetNotificationsForBillPaid -->
  <testcase name="testGetNotificationsForBillPaid" time="45.381" classname="com.yodlee.app.yodleeApi.Insights.Notifications.V2.TestGetInsightsForBillPaid"/>
</testsuite> <!-- Bill Paid -->
